tests/console/journalctl.pm

```diff
@@ -51,7 +56,7 @@ sub verify_journal {
     my $cmd = 'journalctl --verify';
     $cmd .= " --verify-key=$fss_key" if defined($fss_key);
 
-    script_run('journactl --flush');    # ensure data is written to persistent log
+    assert_script_run('journalctl --flush');    # ensure data is written to persistent log
     return if (script_run("$cmd 2>&1 | tee errs") == 0);
```

Comment:
Isn't this always true? Hm no, we have `pipefail` by default right?

Reply:
Tbh, I haven't paid that much attention to this part as I haven't touched this line. :) But the test proceeds with below checks. http://kepler.suse.cz/tests/3826#step/journalctl/212